diff --git a/GQF/mesolve.py b/GQF/mesolve.py
--- a/GQF/mesolve.py
+++ b/GQF/mesolve.py
@@ -10,7 +10,7 @@ states = States()
 
 class MESolve:
     def __init__(self, N=100, batch_size=1, noise_level='low', Hamiltonian='off', chi=None, K_s=None):
-    '''
+        '''
     Class to solve the Lindblad master equation for the dissipative time-evolution. 
 
     PARAMETERS:
@@ -27,7 +27,7 @@ class MESolve:
 
         K_s (float): Value of the Kerr nonlinearity. Specify a value only if the Hamiltonian is 'on' and a different value from the ones specified by the dynamics is desired. Default: None.
 
-    '''
+        '''
         if 'Sivak2023' in Hamiltonian:
             chi, K_s =  2.*3.141592654*0.0465, 2.*3.141592654*4.8*1E-6
         elif 'Ibarcq2020' in Hamiltonian:
